doto-frontend/src/components/pages/Calendar/CalendarListView.js

```diff
@@ -1,7 +1,8 @@
 import React from "react";
 import "./Calendar.css";
 import PropTypes from "prop-types";
-import { Checkbox } from "@material-ui/core";
+import { Checkbox, Typography } from "@material-ui/core";
+import moment from "moment";
```

Comment:
I like the addition of using Moment to display time! Always thought it was implemented inside a scheduling app

Reply:
yeah moment.fromNow() is really cool